packages/clerk-js/src/ui/utils/__tests__/colorMix.spec.ts

```
@@ -0,0 +1,237 @@
+// Mock the cssSupports module
+vi.mock('../cssSupports', () => ({
+  cssSupports: {
+    relativeColorSyntax: vi.fn(),
+    colorMix: vi.fn(),
+  },
+}));
+
+import { afterEach, beforeEach, describe, expect, test, vi } from 'vitest';
```

ESLint

[error] 'afterEach' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)